configs/local.config

```diff
@@ -11,6 +11,7 @@ process {
     withLabel: python3 { cpus = 1 } 
     withLabel: deseq2 { cpus = params.cores } 
     withLabel: fastp { cpus = params.cores } 
+    withLabel: fastqc { cpus = 1 } 
```

Comment:
is fastqc that fast? I'd use it with multiple threads, also with slurm

Reply:
I am running one process for every read (pair) file. 

It's simply what FASTQC is doing anyway if you write

```
fastqc -t 12 *.fastq
```

It will run every fastq file on one thread. 

Or in other words:

```
fastqc -t 24 foo.fastq
```

is not faster compared to

```
fastqc -t 1 foo.fastq
```

And yes, it's fast anyway

Reply:
oh, I didn't know that.
it always felt like `fastqc -t 24 foo.fastq` is faster

Reply:
hm, now I am also not sure... but anyway: only the multiqc process is waiting for the fastqc output so all the other steps will run even when fastqc needs more time. We can increase this later when we see that the whole pipeline is waiting for fastqc